packages/react-ui/src/widgets/FeatureSelectionWidgetUI/FeatureSelectionUIGeometryChips.js

```diff
@@ -52,19 +56,27 @@ function FeatureSelectionUIGeometryChips({
 
   function getFeatureChipLabel(feature, index) {
     const type = translateType(feature.geometry.type);
-    return feature.properties?.name || `${type} ${index + 1}`;
+    return chipLabel || feature.properties?.name || `${type} ${index + 1}`;
   }
 
+  const [onMouseElement, setOnMouseElement] = useState(false);
```

Comment:
Can we give a more meaningful name to this variable ?
Something like `showDeleteTooltip`

Reply:
Fixed https://github.com/CartoDB/carto-react/pull/708/commits/42e895c5efaa33308c60f8179f15229616789a90